migrate_test.go

```diff
@@ -38,6 +47,17 @@ func init() {
 	sourceStubMigrations.Append(&source.Migration{Version: 5, Direction: source.Down, Identifier: "DROP 5"})
 	sourceStubMigrations.Append(&source.Migration{Version: 7, Direction: source.Up, Identifier: "CREATE 7"})
 	sourceStubMigrations.Append(&source.Migration{Version: 7, Direction: source.Down, Identifier: "DROP 7"})
+
+	sourceMigrationTaskMigrations = source.NewMigrations()
+	sourceMigrationTaskMigrations.Append(&source.Migration{Version: 1, Direction: source.Up, Identifier: "CREATE 1"})
+	sourceMigrationTaskMigrations.Append(&source.Migration{Version: 1, Direction: source.Down, Identifier: "DROP 1"})
+	sourceMigrationTaskMigrations.Append(&source.Migration{Version: 2, Direction: source.Up, Identifier: ""})
+	sourceMigrationTaskMigrations.Append(&source.Migration{Version: 2, Direction: source.Down, Identifier: ""})
+	sourceMigrationTaskMigrations.Append(&source.Migration{Version: 3, Direction: source.Up, Identifier: "CREATE 3"})
+	sourceMigrationTaskMigrations.Append(&source.Migration{Version: 4, Direction: source.Up, Identifier: "CREATE 4"})
+	sourceMigrationTaskMigrations.Append(&source.Migration{Version: 4, Direction: source.Down, Identifier: "DROP 4"})
+	sourceMigrationTaskMigrations.Append(&source.Migration{Version: 5, Direction: source.Down, Identifier: "DROP 5"})
```

Comment:
Might be worth mentioning in a comment here that the "tasks" are actually defined in the unit test function.